File: two-sum/go/bruteforce.go
package main

func TwoNumberSum(array []int, target int) []int {
	// Write your code here.
	result := []int{}
	for i, n1 := range array {
		for j, n2 := range array {
			if (i != j && n1 + n2 == target){
				result = append(result, n1)
				result = append(result, n2)
				return result
			}
		}
	}
	return result
}

File: two-sum/java/bruteforce.java
import java.util.*;

class Program {
  public static int[] twoNumberSum(int[] array, int targetSum) {
    // Write your code here.    
    for(int i = 0; i<array.length; i++){
      for(int j = 0; j<array.length; j++){
        if(i != j){
          int tempSum = array[i] + array[j];
          if(tempSum == targetSum){
            return new int[]{array[i], array[j]};
          }
        }
      }
    }
    return new int[0];
  }
}
